cirq-google/cirq_google/engine/engine_validator.py

```diff
@@ -73,7 +72,6 @@ def validate_program(
     max_size: int = MAX_MESSAGE_SIZE,
 ) -> None:
     """Validate that the Program message size is below the maximum size limit.
-
     Args:
         circuits:  A sequence of  `cirq.Circuit` objects to validate.  For
```

Comment:
Should `circuits` be a singular `circuit` and not a `Sequence` now?

Reply:
This gets into the realm of exploding file changes since there are many apis which validate their circuit and account for the api accepting a Sequence by passing `[circuit]` or `[program]` for regular `program_pb2.Program`s . 

I agree this should happen eventually, so I've added a TODO to alter the parameter to no longer accept Sequences.

Reply:
SGTM, thanks!